packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -367,6 +367,8 @@ export class AnchoredRegion extends FASTElement {
     private pendingReset: boolean = false;
     private currentDirection: Direction = Direction.ltr;
 
+    private static intersectionService: IntersectionService = new IntersectionService();
```

Comment:
Why not remove the association with anchored region and instead export from the `./intersection-service` a new instance that any component can import? That way this can be re-usable for other components.

Reply:
I've considered that but figured we'd get it in as a private thing before we add it to our public api.  If folks think we should make it public from the start I can make that change.